android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/surveyquestions/SurveyQuestionsScreen.kt

```diff
@@ -217,6 +228,36 @@ private fun SurveyQuestionsContent(questions: List<QuestionContentUiModel>) {
     }
 }
 
+@Composable
+private fun Answer(type: QuestionDisplayType, onAnswerClick: (Int) -> Unit) {
+    when (type) {
+        QuestionDisplayType.STAR -> EmojiRatingAnswer(
+            emojis = MutableList(5) {
```

Comment:
same

```suggestion
            emojis = List(5) {
```

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/116/commits/5823aa07c1c30a1dc9437a8f40ad7c587c2f3997